### PR TITLE
852 ElastiSearch Schema updates for Sighting, Encounter and Individual

### PR DESCRIPTION
**Pull request by hwindsor**
## Pull Request Overview

- Populate ElasticSearch Schema for Sighting, Encounter and Individual from EDM data
- New util addes in FeatherModel to access an individual field

---

### CODE REVIEW

File: app/modules/encounters/models.py [hunk 1]
```diff
@@ -117,19 +116,39 @@ def __repr__(self):
...
+            location_id = self.sighting.get_location_id()
         return location_id
 
+    def get_location(self):
+        return self.get_edm_data_field('locationId')
+
+    def get_point(self):
+        lat = self.get_edm_data_field('decimalLatitude')
+        long = self.get_edm_data_field('decimalLongitude')
+        return f'{lat},{long}' if lat and long else None
```
Comment (hwindsor) [inline app/modules/encounters/models.py:142]: This is a big guess about what is m eant by 'point'

File: app/modules/individuals/schemas.py [hunk 2]
```diff
@@ -81,7 +84,11 @@ class ElasticsearchIndividualSchema(ModelSchema):
         attribute='names',
         many=True,
     )
+    encounters = base_fields.Nested(ElasticsearchEncounterSchema, many=True)
```
Comment (hwindsor) [inline app/modules/individuals/schemas.py:87]: To me it made most sense to use the Encounter ES schema to keep a consistent output :man_shrugging:

File: app/modules/encounters/models.py (hunk 1 above)
Comment (naknomum) [reply app/modules/encounters/models.py:145]: my reading of [these docs](https://www.elastic.co/guide/en/elasticsearch/reference/current/geo-point.html) is that this is a valid format.  yet: these docs seem to swap (!!!) lat/lon for lon/lat on some of these formats.  what???   the string-version though appears to be `lat,lon`.   (unlike the _array_ and _WKT POINT_ for some reason!)

**Review by naknomum — APPROVED** [review]
looks good to me.

**Pull request merged**